src/components/Flashcard/Flashcard.js

```diff
@@ -1,100 +1,132 @@
         if (tag === 'known') {
-            addingAndRemovingFromBase(id, known,setKnown, unknown, setUnknown);
-        }else{
+            addingAndRemovingFromBase(id, known, setKnown, unknown, setUnknown);
+        } else {
             addingAndRemovingFromBase(id, unknown, setUnknown, known, setKnown);
         }
         checkAllWithUserBase(questions);
-    }
+    };
 
-    const addingAndRemovingFromBase = (id, baseToAdd, setBaseToAdd, baseToRemove, setBaseToRemove ) => {
-        if(!baseToAdd.includes(id)){
+    const addingAndRemovingFromBase = (
+        id,
+        baseToAdd,
+        setBaseToAdd,
+        baseToRemove,
+        setBaseToRemove
+    ) => {
+        if (!baseToAdd.includes(id)) {
             let newBaseToAdd = baseToAdd;
             newBaseToAdd.push(id);
             setBaseToAdd(newBaseToAdd);
-            if(baseToRemove.includes(id)){
-                let newBaseToRemove = baseToRemove.filter(item => item !== id);
+            if (baseToRemove.includes(id)) {
+                let newBaseToRemove = baseToRemove.filter((item) => item !== id);
                 setBaseToRemove(newBaseToRemove);
             }
         }
-    }
+    };
 
     const nextQuestion = () => {
-        if(countQuestions === previousQuestions.length){
+        if (countQuestions === previousQuestions.length) {
             drawNextQuestion();
-        }else{
+        } else {
             let forward = +1;
             movingInsidePreviousQuestions(forward);
         }
-    }
+    };
 
     const backToPrevious = () => {
         let backward = -1;
         movingInsidePreviousQuestions(backward);
-    }
+    };
 
     const movingInsidePreviousQuestions = (direction) => {
         let newCount = countQuestions + direction;
-        let newCurrentQuestion = questions.find(question => question.id === previousQuestions[newCount - 1]);
+        let newCurrentQuestion = questions.find(
+            (question) => question.id === previousQuestions[newCount - 1]
+        );
         setCurrentQuestion(newCurrentQuestion);
         setCountQuestions(newCount);
-    }
+    };
 
     const drawNextQuestion = () => {
         let nextId;
-        while(nextId === undefined || previousQuestions.includes(nextId)){
-            let randomCategory = Math.floor(Math.random() * 100); 
+        while (nextId === undefined || previousQuestions.includes(nextId)) {
+            let randomCategory = Math.floor(Math.random() * 100);
             if (randomCategory < 45) {
                 nextId = randomIdFromCategory(unknown);
-            }else if(randomCategory < 85){
+            } else if (randomCategory < 85) {
                 nextId = randomIdFromCategory(all);
-            }else{
+            } else {
                 nextId = randomIdFromCategory(known);
             }
         }
-        let nextQuestion = questions.find(question => question.id === nextId);
+        let nextQuestion = questions.find((question) => question.id === nextId);
```

Comment:
is this bracket necessary?

Reply:
nope, I changed that (its a prettier default setting I'll look for it later)